### DISCUSSION TITLE
关于是否应该默认打开请求时携带Cookie

### DISCUSSION THREAD

**Discussion by timi137137** [Tencent/tdesign-vue-next-starter#528] [Ideas]
本讨论用于讨论
模板项目中是否应该默认打开请求时携带Cookie

**Comment by PDieE** [comment]
我的建议是默认关闭，原因如下：
1. 目前已经前后端分离了，我想大部分前后端分离的系统如果没有特殊要求的话，都是通过 请求的 header 或者body 传输数据或鉴权数据什么的
2. 如上所说，如果某系统要求携带cookies的话，那么使用者要么本身会懂，就算不懂的话也会去搜索相关内容，且这个问题也很好描述所以很好获取大胆
3. 如果说系统确实不需要携带cookies的话，那么默认携带cookies可能会造成一些意外问题，比如跨域，而且现今的开发人员能力层次不起，甚至可以说这个金字塔底端的人群其实是很多的，他就会很郁闷很懵
